theseus_gui/src/components/RowDisplay.vue

```diff
@@ -17,58 +31,189 @@ const props = defineProps({
   canPaginate: Boolean,
 })
 
-const allowPagination = ref(false)
+const allowPagination = ref(Array.apply(false, Array(props.instances.length)))
 const modsRow = ref(null)
+const instanceOptions = ref(null)
+const instanceComponents = ref(null)
 
 const handlePaginationDisplay = () => {
-  let parentsRow = modsRow.value
-
-  // This is wrapped in a setTimeout because the HtmlCollection seems to struggle
-  // with getting populated sometimes. It's a flaky error, but providing a bit of
-  // wait-time for the below expressions has not failed thus-far.
-  setTimeout(() => {
-    const children = parentsRow.children
-    const lastChild = children[children.length - 1]
-    const childBox = lastChild?.getBoundingClientRect()
-
-    if (childBox?.x + childBox?.width > window.innerWidth && props.canPaginate)
-      allowPagination.value = true
-    else allowPagination.value = false
-  }, 300)
+  for (let i = 0; i < props.instances.length; i++) {
+    let parentsRow = modsRow.value[i]
+
+    // This is wrapped in a setTimeout because the HtmlCollection seems to struggle
+    // with getting populated sometimes. It's a flaky error, but providing a bit of
+    // wait-time for the below expressions has not failed thus-far.
+    setTimeout(() => {
+      const children = parentsRow.children
+      const lastChild = children[children.length - 1]
+      const childBox = lastChild?.getBoundingClientRect()
+
+      if (childBox?.x + childBox?.width > window.innerWidth && props.canPaginate)
+        allowPagination.value[i] = true
+      else allowPagination.value[i] = false
+    }, 300)
+  }
```

Comment:
why was this changed to do it for each instance? i don't think this would do anything